src/main/kotlin/snyk/common/lsp/LanguageServerWrapper.kt

```diff
@@ -214,7 +217,12 @@ class LanguageServerWrapper(
             logger.warn("No active project found, not sending scan command.")
             return
         }
-        project.getContentRootVirtualFiles().mapNotNull { it.path }.toSet().forEach(::sendFolderScanCommand)
+        project.getContentRootVirtualFiles().filterNotNull().toSet().forEach {
+            val trustService = service<WorkspaceTrustService>()
+            if (trustService.isPathTrusted(it.toNioPath())) {
+                sendFolderScanCommand(it.path)
+            }
+        }
```

Comment:
Checking what `toNioPath` I saw that it can `throw` so I wonder if we need to handle the error.
<img width="738" alt="image" src="https://github.com/snyk/snyk-intellij-plugin/assets/1948377/762505d5-dd0d-471d-945d-dbc528f59c11">
It seems that we can `try/catch` it?

Reply:
This should not be possible to happen - but yeah, I'll make a note to include a catch with a user notification in the next PR.